<ui version="4.0" >
 <author>Copyright (C) 2006-2010 Felix Geyer</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>WebMonX</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>44</y>
     <width>260</width>
     <height>306</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <property name="spacing" >
     <number>6</number>
    </property>
    <property name="margin" >
     <number>9</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>260</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools" >
    <property name="title" >
     <string>&amp;Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuPage" >
    <property name="title" >
     <string>&amp;Page</string>
    </property>
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuPage" />
   <addaction name="menuTools" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>350</y>
     <width>260</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>29</y>
     <width>260</width>
     <height>15</height>
    </rect>
   </property>
   <property name="movable" >
    <bool>false</bool>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
